cantor-misc/src/main/java/com/salesforce/cantor/misc/archivable/AbstractBaseArchivableNamespaceable.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020, Salesforce.com, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package com.salesforce.cantor.misc.archivable;
+
+import com.salesforce.cantor.Namespaceable;
+
+import java.io.IOException;
+import java.util.Collection;
+
+abstract class AbstractBaseArchivableNamespaceable<D extends Namespaceable, T extends EventsArchiver>
```

Comment:
`<T extends Namespaceable, A extends CantorArchiver>` instead of `EventsArchiver`

Reply:
Oh yep, missed that. Fixed. Also renamed `Archiver` to `CantorArchiver`. More in-line with the `Cantor` interface design.